sklearn/manifold/tests/test_isomap.py

```diff
@@ -186,3 +231,13 @@ def test_sparse_input():
                                   eigen_solver=eigen_solver,
                                   path_method=path_method)
             clf.fit(X)
+
+
+def test_isomap_fit_precomputed_radius_graph():
+    X, _ = datasets.load_digits(return_X_y=True)
+    X = preprocessing.StandardScaler().fit_transform(X)
+    g = neighbors.radius_neighbors_graph(X, radius=5.5, mode="distance")
+    isomap = manifold.Isomap(n_neighbors=None,
+                             radius=5.5,
+                             metric="precomputed")
+    isomap.fit(g)
```

Comment:
Could you please extend this test to check that the output of `fit_transform` with precomputed radius graph is (approximately) the same as the output  `fit_transform` without precomputing the graph?